### PR TITLE
Fixed type error when listing sound cards

### PR DESCRIPTION
**Pull request by EvTheFuture**

### CODE REVIEW

File: packages/addons/service/snapclient/source/addon.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 card = ''
 cards = []
-lines = subprocess.check_output([SNAPCLIENT, '--list']).splitlines()
+lines = lines = subprocess.getoutput(SNAPCLIENT + ' --list').splitlines()
```
Comment (vpeter4) [inline packages/addons/service/snapclient/source/addon.py]: Why `lines = lines = ` ?

Also you can send PR to upstream project (LibreELEC).

**Comment by vpeter4** [comment]
You can send PR to upstream project (LibreELEC).

**Comment by EvTheFuture** [comment]
> You can send PR to upstream project (LibreELEC).

PR sent to LibreELEC as well

**Comment by EvTheFuture** [comment]
PR for LibreELEC [HERE](https://github.com/LibreELEC/LibreELEC.tv/pull/5290)

**Comment by vpeter4** [comment]
Thank you.

**Pull request closed without merging**